package.json

```diff
@@ -20,5 +20,8 @@
     "jest": "^29.0.1",
     "jest-environment-jsdom": "^29.0.1",
     "sinon-chrome": "^3.0.1"
+  },
+  "dependencies": {
+    "punycode": "^2.1.1"
```

Comment:
Can you explain the need for this paclage?

Reply:
Under the hood, JS uses two character encoding standards. One is (I think) UTF-16, and the other is UCS2, which is a standard from the early days of Unicode.
`charCodeAt` and `String.fromCodePoint` use UCS2, and thus apparently don't support later Unicode characters (`<boldLetterString`.charCodeAt(0)` returns `55349`, which is the unicode character for "unknown character").

While we don't need all of `punycode`, I use it's UTF-16 <-> UCS2 capabilities to get bold characters' values for the needed arithmetics.